impl/src/test/java/org/ehcache/docs/UserManagedCaches.java

```diff
@@ -21,20 +21,14 @@
 import org.ehcache.config.builders.CacheEventListenerConfigurationBuilder;
 import org.ehcache.config.builders.ResourcePoolsBuilder;
 import org.ehcache.config.builders.UserManagedCacheBuilder;
-import org.ehcache.impl.config.persistence.DefaultPersistenceConfiguration;
-import org.ehcache.event.EventType;
-import org.ehcache.impl.config.persistence.UserManagedPersistenceContext;
 import org.ehcache.config.units.EntryUnit;
 import org.ehcache.config.units.MemoryUnit;
+import org.ehcache.core.spi.service.LocalPersistenceService;
 import org.ehcache.docs.plugs.ListenerObject;
-import org.ehcache.docs.plugs.LongCopier;
+import org.ehcache.event.EventType;
+import org.ehcache.impl.config.persistence.DefaultPersistenceConfiguration;
+import org.ehcache.impl.config.persistence.UserManagedPersistenceContext;
```

Comment:
It's related. A bit. Modifying StringCopier made me revise where it is used. And here was a unused import.